spec/helpers/sessions_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe SessionsHelper, type: :helper do
+  let(:user) do
+    User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+  end
+
+  describe '#log_in' do
+    subject { log_in(user) }
+    let(:logged_in_user) { user.id }
+    it { is_expected.to be logged_in_user }
+  end
+
+  describe '#current_user' do
+    subject { current_user }
+    let(:logged_in_user) { user }
+    before { log_in(user) }
+    it { is_expected.to eq logged_in_user }
+  end
+
+  describe '#log_out' do
+    subject { log_out }
+    before do
+      log_in(user)
+      current_user
+    end
+    it { expect { subject }.to change { session[:user_id] } }
```

Comment:
This example is checking more than just a simple return value, so I would prefer a better description on the example, ie:
```
it "sets the current user id in the session" do
  ...
end
```